### ISSUE TITLE
Is there a way to fine-tune/train the model, with a couple of hundreds of examples?

### ISSUE THREAD

**Issue by BlackLight367437** [Azure-Samples/azure-search-openai-demo#1014]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Versions
> GPT 3.5

### Mention any other details that might be useful

Hi,
We've got a couple of hundreds of sample questions and sample answers, which would be a great standard for the ChatBot to follow when providing answers.
We obviously can't fit them into a prompt, because of the maximum number of tokens limitations.
Is there a way to fine-tune/train the current GPT 3.5 (or GPT 4) models with those samples?

I mean something like this:
https://openai.com/blog/customizing-gpt-3
https://platform.openai.com/docs/guides/fine-tuning

If fine-tuning/training would be possible, what's the proper way of doing it?
(Should we just follow this documentation: https://platform.openai.com/docs/guides/fine-tuning ? Or is it a different process?)

Thank you in advance for the answer and best regards.

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pamelafox** [comment]
Yes, you should be able to fine-tune. We have Azure specific documentation here:
https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/fine-tuning?tabs=turbo&pivots=programming-language-studio

We often say that fine-tuning is a last resort, due to the cost of fine tuning, the cost of using a fine-tuned model, and the often negligible benefits, but that's a decision you can make when you see how performance improves.  You can look at my PR on evaluation (https://github.com/Azure-Samples/azure-search-openai-demo/pull/967) for an idea of how to evaluate the results to see how much improved they are. If you haven't already, I recommend you first try changing the prompt and adding some examples (few shots), before going down the fine tuning route.

Let us know how it goes!
